SPECS/build-essential/build-essential.spec

```diff
@@ -13,7 +13,7 @@ Requires:       gawk
 Requires:       gcc
 Requires:       glibc-devel
 Requires:       gzip
-Requires:       installkernel
+Requires:       systemd-udev
```

Comment:
This causes a build cycle: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=509975&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=88a626d5-bd2d-5bcb-0f14-68ce5ab12667&l=4331

Reply:
Yeah, it seems that installkernel was only added to build-essential as a convenience to the user. So, it is probably fine to remove.